# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchDriver.h
# Commit: 7f9255188d00
# Full Hash: 7f9255188d00c8f0cb4338d4604602111378de59
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-27 09:40:54
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Getting PerformanceTimingData from channel for FetchService. r=dom-worker-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138249
# ==============================================================================

diff -r 3eecad934fc1 -r 7f9255188d00 dom/fetch/FetchDriver.h
--- a/dom/fetch/FetchDriver.h	Sat Feb 26 23:24:47 2022 +0000
+++ b/dom/fetch/FetchDriver.h	Sun Feb 27 00:26:21 2022 +0000
@@ -38,6 +38,7 @@
 class InternalRequest;
 class InternalResponse;
 class PerformanceStorage;
+class PerformanceTimingData;
 
 /**
  * Provides callbacks to be called when response is available or on error.
@@ -124,6 +125,9 @@
     mOriginStack = std::move(aOriginStack);
   }
 
+  PerformanceTimingData* GetPerformanceTimingData(nsAString& aInitiatorType,
+                                                  nsAString& aEntryName);
+
   // AbortFollower
   void RunAbortAlgorithm() override;
 